src/scss/components/button/_index.scss

```diff
@@ -1,3 +1,14 @@
+@import "vars";
```

Comment:
Import is unnecessary. There are no variables being consumed directly in this file.